src/main/java/net/fabricmc/loader/impl/gui/FabricMainWindow.java

```diff
@@ -89,11 +92,29 @@ private static void open0(FabricStatusTree tree, boolean shouldWait) throws Exce
 		}
 	}
 
+	private static String getTitleVersion() {
```

Comment:
Player asks to move this method below the caller `createUi`. Guess things are good beyond that.

Reply:
Fixed in f4a646d8cddc5a269db2108f37e1d2a7fc825e0d